### PR TITLE
Add myself to CODEOWNERS

### PR DESCRIPTION
**Pull request by jb3**
Add myself, Joe, to the CODEOWNERS.

### CODE REVIEW

**Review by mbaruh — APPROVED** [review]
Borderline, but whatever

File: .github/CODEOWNERS [hunk 1]
```diff
@@ -4,14 +4,14 @@
...
-bot/exts/utils/snekbox.py               @MarkKoz @Akarys42
+bot/exts/utils/snekbox.py               @MarkKoz @Akarys42 @jb3
 bot/exts/help_channels/**               @MarkKoz @Akarys42
-bot/exts/moderation/**                  @Akarys42 @mbaruh @Den4200 @ks129
-bot/exts/info/**                        @Akarys42 @Den4200
-bot/exts/info/information.py            @mbaruh
-bot/exts/filters/**                     @mbaruh
+bot/exts/moderation/**                  @Akarys42 @mbaruh @Den4200 @ks129 @jb3
+bot/exts/info/**                        @Akarys42 @Den4200 @jb3
+bot/exts/info/information.py            @mbaruh @jb3
```
Comment (mbaruh) [inline .github/CODEOWNERS:11]: Why did you add yourself to both the folder and the file? lol

**Review by ChrisLovering — APPROVED** [review]
